crates/sqlexec/src/remote/client.rs

```diff
@@ -193,14 +186,33 @@ impl RemoteClient {
         }
 
         let mut dst: Endpoint = dst.try_into()?;
-        if let Some(tls_conf) = tls_conf {
-            metadata.insert(CA_CERT_PATH, tls_conf.ca_cert_path.parse()?);
-            metadata.insert(DOMAIN, tls_conf.domain.parse()?);
-            let ca = std::fs::read_to_string(tls_conf.ca_cert_path)?;
+
+        let mut api_url = "https://console.glaredb.com/api/internal/authenticate/client";
+        let host = dst.uri().host().expect("invalid host");
+        if host.contains("qa.glaredb.com") {
+            api_url = "https://qa.glaredb.com/api/internal/authenticate/client";
+        }
```

Comment:
This could be written like:

```rust
let host = dst.uri().host()?;
let api_url = if host.contains("qa.glaredb.com") {
    "https://qa.glaredb.com/api/internal/authenticate/client"
} else {
    "https://console.glaredb.com/api/internal/authenticate/client"
}

```